minimumFallingPathSum.cpp

```diff
@@ -0,0 +1,34 @@
+//time complexity: O(n^2)
+//space complexity:O(n)
+//approach: brute force
+//solved on leetcode? yes, but only 39/69 test cases pass
+//problem faced? yes, I was able to think of the approach but was unable to code it so I saw the solution.
+class Solution {
```

Comment:
Try the recursion approach first and then try the DP approach as taught in class. Let me know if you still have questions.